### PR TITLE
chore: Re-enable e2e tests, add cron schedule to build definition

### PR DESCRIPTION
**Pull request by k-rush**
## What changes are proposed in this pull request?

Re-enable e2e tests and add cron trigger for main build pipeline

## How is this patch tested?

- [ ] I have written tests (not required for typo or doc fix) and confirmed the proposed feature/bug-fix/change works.

## Does this PR change any dependencies?

- [x] No. You can skip this section.
- [ ] Yes. Make sure the dependencies are resolved correctly, and list changes here.

## Does this PR add a new feature? If so, have you added samples on website?

- [x] No. You can skip this section.
- [ ] Yes. Make sure you have added samples following below steps.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [templates/get_cst_notebooks.yml](https://github.com/microsoft/SynapseML/blob/3374e95dfc52a56515aac7bdab37096655a48918/templates%2Fget_cst_notebooks.yml) | 100 | 100 | 100 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/867b9445-4089-4ed7-9fc2-85120d183a4c) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: core/src/test/scala/com/microsoft/azure/synapse/ml/Secrets.scala [hunk 1]
```diff
@@ -59,14 +59,27 @@ object Secrets {
...
+  lazy val SynapseExtensionDailyTenantId: String = getSecret("synapse-extension-daily-tenant-id")
+  lazy val SynapseExtensionDailyUxHost: String = getSecret("synapse-extension-daily-ux-host")
+  lazy val SynapseExtensionDailySspHost: String = getSecret("synapse-extension-daily-ssp-host")
+  lazy val SynapseExtensionDailyWorkspaceId: String = getSecret("synapse-extension-daily-workspace-id")
+
+  lazy val SynapseExtensionDxtPassword: String = getSecret("synapse-extension-dxt-password")
+  lazy val SynapseExtensionDxtTenantId: String = getSecret("synapse-extension-dxt-tenant-id")
+  lazy val SynapseExtensionDxtUxHost: String = getSecret("synapse-extension-dxt-ux-host")
+  lazy val SynapseExtensionDxtSspHost: String = getSecret("synapse-extension-dxt-ssp-host")
+  lazy val SynapseExtensionDxtWorkspaceId: String = getSecret("synapse-extension-dxt-workspace-id")
```
Comment (mhamilton723) [inline core/src/test/scala/com/microsoft/azure/synapse/ml/Secrets.scala:84]: Theres a bit of duplication caused by this, perhaps we can make this more programmatic by specifying the deployment (dev, daily, weekly, prod) in a variable in this file and only getting secrets for that version this way we can reduce the repeated code and make the var names a bit shorter

**Review by mhamilton723 — APPROVED** [review]

**Review by svotaw — APPROVED** [review]

**Pull request merged**